gradle/errorprone/warnings

```diff
@@ -62,7 +62,6 @@ errorProneWarnings = \
 -Xep:ReferenceEquality:ERROR,\
 -Xep:RequiredModifiers:ERROR,\
 -Xep:ShortCircuitBoolean:ERROR,\
--Xep:SimpleDateFormatConstant:ERROR,\
```

Comment:
Why remove this?

Reply:
It was removed:
> Task :opencensus-api:compileJava FAILED
com.google.errorprone.InvalidCommandLineOptionException: SimpleDateFormatConstant is not a valid checker name